client/wp-admin-scripts/print-shipping-label-banner/dismiss-modal/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { Button, Modal } from '@wordpress/components';
+import { withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import '../style.scss';
+
+export class DismissModal extends Component {
+	setDismissed = ( timestamp ) => {
+		this.props.updateOptions( {
+			woocommerce_shipping_dismissed_timestamp: timestamp,
+		} );
+	};
+
+	hideBanner = () => {
+		document.getElementById(
+			'woocommerce-admin-print-label'
```

Comment:
Any chance this element might not be in the DOM, should we check just in case?

Reply:
No, not really. If it's not in the dom then we could never get here.